core/src/net/sf/openrocket/rocketcomponent/FinSet.java

```diff
@@ -885,24 +884,25 @@ public boolean isRootStraight( ){
 		
 		final double xTabFront = getTabFrontEdge();
 		final double xTabTrail = getTabTrailingEdge();
+
+		// // limit the new heights to be no greater than the current body radius.
+		// this.tabHeight = Math.min( this.tabHeight,  bodyRadius );
```

Comment:
Can remove commented out code added here, unless you think its intended to add something?

Reply:
yea, removed.